### PR TITLE
Added CMC and Coingecko activities

### PR DESCRIPTION
**Pull request by antonionardella**
# Description of change

Added CoinMarketCap and Coingecko activities to the how-to-support page

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Enhancement

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [X] I have followed the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: internal/learn/the-community/how-to-support.md [hunk 1]
```diff
@@ -9,27 +9,55 @@ As ecosystem members there are different ways to support the vision.
...
 
-#### Like and retweet news in the ecosystem.
+Like and retweet news in the ecosystem.
 
 - Difficulty level: **easy**
 - Time needed: **15 minutes/week**
 - Requirements: [Sign up to Twitter](https://twitter.com/i/flow/signup)
 
-Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow them daily to *like* and *retweet*. Look for other projects or ecosystem members, follow them, and like their tweets.
+Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow them daily to _like_ and _retweet_. Look for other projects or ecosystem members, follow them, and like their tweets.
```
Comment (Dr-Electron) [inline internal/learn/the-community/how-to-support.md:60]:
```suggestion
Follow the [IOTA Foundation](https://twitter.com/iota/) and [Shimmer](https://twitter.com/shimmernet) on Twitter and follow them daily to *like* and *retweet*. Look for other projects or ecosystem members, follow them, and like their tweets.
```

File: internal/learn/the-community/how-to-support.md [hunk 2]
```diff
@@ -41,9 +69,8 @@ Like and reshare news from the IOTA Foundation and other ecosystem members with
 - Time needed: **15 minutes/week**
 - Requirements: [Sign up to LinkedIn](https://www.linkedin.com/signup/cold-join). You can also connect using your Google Account.
 
-Follow the IOTA Foundation on LinkedIn [here](https://www.linkedin.com/company/iotafoundation/) by clicking on the three dots and then **+ Follow** to *like* and *share* the articles
-
-![image-20210115161351884](/img/participate/how-to-support/image-20210115161351884.png)
+Follow the IOTA Foundation on LinkedIn [here](https://www.linkedin.com/company/iotafoundation/) by clicking on the three dots and then **+ Follow** to _like_ and _share_ the articles
```
Comment (Dr-Electron) [inline internal/learn/the-community/how-to-support.md:72]:
```suggestion
Follow the IOTA Foundation on LinkedIn [here](https://www.linkedin.com/company/iotafoundation/) by clicking on the three dots and then **+ Follow** to *like* and *share* the articles
```

File: internal/learn/the-community/how-to-support.md [hunk 3]
```diff
@@ -64,9 +91,8 @@ Starring a repository has some potentially beneficial effects for the repository
 - Time needed: **30 minutes the first time, 5 minutes for every new repository (when they are published)**.
 - Requirements: [Sign up to GitHub](https://github.com/join).
 
-After you have fulfilled the requirements, open the following repositories and *star* them:
-
-![image-20210115162757470](/img/participate/how-to-support/image-20210115162757470.png)
+After you have fulfilled the requirements, open the following repositories and _star_ ⭐ them:
```
Comment (Dr-Electron) [inline internal/learn/the-community/how-to-support.md:94]:
```suggestion
After you have fulfilled the requirements, open the following repositories and *star* ⭐ them:
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]
You need to undo the yarn change. No devs update should be needed for this PR.
What you can then do, to check the wiki, is the following:
Update/Init all submodules:
`git submodule update --init --recursive`
Instapp deps
`yarn`
Optional: Start wiki
`yarn start:all`

File: internal/learn/the-community/how-to-support.md [hunk 4]
```diff
@@ -9,21 +9,49 @@ As ecosystem members there are different ways to support the vision.
...
+## Crypto platforms
+
+### Coinmarketcap
+
+#### Watchlist
+
+Register an account or use your existing Binance.com account.
+Add [IOTA](https://coinmarketcap.com/currencies/iota/) and [Shimmer](https://coinmarketcap.com/currencies/shimmer/) to your watchlist by clicking on the ⭐
+
+![coinmarketcap_iota_watchlist.png](/img/participate/how-to-support/coinmarketcap_iota_watchlist.png)
```
Comment (Dr-Electron) [inline internal/learn/the-community/how-to-support.md:25]: I think for SEO it is better to use a descriptive text. Also for accessibility. Maybe you want to change that 🤔

File: internal/learn/the-community/how-to-support.md [hunk 5]
```diff
@@ -64,9 +91,8 @@ Starring a repository has some potentially beneficial effects for the repository
 - Time needed: **30 minutes the first time, 5 minutes for every new repository (when they are published)**.
 - Requirements: [Sign up to GitHub](https://github.com/join).
 
-After you have fulfilled the requirements, open the following repositories and *star* them:
-
-![image-20210115162757470](/img/participate/how-to-support/image-20210115162757470.png)
+After you have fulfilled the requirements, open the following repositories and *star* ⭐ them:
+![image-20210115162757470](/img/participate/how-to-support/github_iota_star.png)
```
Comment (Dr-Electron) [inline internal/learn/the-community/how-to-support.md:95]: At least you should change the text here ;)

**Review by Dr-Electron — APPROVED** [review]
Looks good. Just some small hints about [image](https://wiki.iota.org/learn/contribute-to-wiki/reference/style/formatting#images) descriptions

**Review by antonionardella — COMMENTED** [review]
Solved suggestions

**Pull request merged**